app/_src/production/cp-deployment/multi-zone.md

```diff
@@ -15,7 +15,23 @@ To set up a multi-zone deployment we will need to:
 ## Usage
 ### Set up the global control plane
 
+{% if_version gte:2.2.x %}
+The global control plane must run on a dedicated cluster (unless using "Universal on Kubernetes" mode), and cannot be assigned to a zone.
+
+{% tip %}
+
+Running global control plane in "Universal on Kubernetes" mode means using PostgreSQL as storage instead of Kubernetes.
+It means that failover / HA / reliability characteristics will change.
+Please read [Kubernetes](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/ha-topology/) and
+[PostgreSQL](https://www.postgresql.org/docs/current/high-availability.html) for more details.
+
+{% endtip %}
```

Comment:
@jakubdyszkiewicz can you take a look here. WDYT is better, have this tip inside the tab or outside like it is right now?